lib/stylelint/10.0.x/stylelint.config.js

```diff
@@ -0,0 +1,436 @@
+module.exports = {
+	defaultSeverity: "error",
+	rules: {
```

Comment:
Can you add the rule [no-descending-specificity](https://github.com/stylelint/stylelint/blob/master/lib/rules/no-descending-specificity/README.md) also here as you did in the config for v11?

Because that rule is available since version 4.5.0 😉